### PR TITLE
level.js -> level-js

### PR DESCRIPTION
**Pull request by ralphtheninja**
Closes https://github.com/Level/level-js/issues/120

### CODE REVIEW

File: collaborators.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Collaborators
 
-level.js is only possible due to the excellent work of the following collaborators:
+`level-js` is only possible due to the excellent work of the following collaborators:
```
Comment (ralphtheninja) [inline collaborators.md]: @vweevers Should we keep this file at all? Merge it with README? Or link to `level/community`?

Comment (ralphtheninja) [reply collaborators.md]: Oh we already are linking from README to `level/community`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -57,7 +57,7 @@
 * Rely on `abstract-leveldown` defaults in `Iterator` constructor (@vweevers)
 * Rely on `abstract-leveldown` callback defaults (@vweevers)
 * Remove testling from `package.json` (@vweevers)
-* Remove `level.js` logo (@vweevers)
+* Remove `level-js` logo (@vweevers)
```
Comment (vweevers) [inline CHANGELOG.md]: Actually I purposefully called it `level.js` because that's what the logo said ;) But doesn't matter

Comment (ralphtheninja) [reply CHANGELOG.md]: Just made a query replace. I'll fix.

File: collaborators.md (hunk 1 above)
Comment (vweevers) [reply collaborators.md]: We do need a standard solution across the board. The contributors section of `level/community` is slightly outdated, and maybe it'd be better to list contributors per project?

Comment (vweevers) [reply collaborators.md]: Can do that later though.

Comment (ralphtheninja) [reply collaborators.md]: Yes lets do this.

**Pull request merged**